src/sentry/testutils/cases.py

```diff
@@ -699,20 +699,6 @@ def assertPluginInstalled(self, name, plugin):
                 )
         self.fail("Missing plugin from entry_points: %r" % (name,))
 
-    # TODO (Steve): remove function
-    def assertTestOnlyPluginInstalled(self, name, plugin):
-        path = type(plugin).__module__ + ":" + type(plugin).__name__
-        for ep in iter_entry_points("sentry.test_only_plugins"):
-            if ep.name == name:
-                ep_path = ep.module_name + ":" + ".".join(ep.attrs)
-                if ep_path == path:
-                    return
-                self.fail(
-                    "Found plugin in entry_points, but wrong class. Got %r, expected %r"
-                    % (ep_path, path)
-                )
-        self.fail("Missing plugin from entry_points: %r" % (name,))
-
     # TODO (Steve): remove function
     def assertNewPluginInstalled(self, name, plugin):
```

Comment:
Is this gonna be removed in a follow-up pr after this is merged?

Reply:
@joshuarli Yes. In the follow-up PR I will move everything from `new_sentry_plugins` into `sentry_plugins` and clean up other items.

Reply:
🎉